src/appengine/handlers/cron/grouper.py

```diff
@@ -200,6 +203,17 @@ def _group_testcases_based_on_variants(testcase_map):
                 testcase_2.crash_type, testcase_2.crash_state,
                 testcase_2.security_flag, testcase_2.group_id))
 
+      top_matched_testcase = sorted(
+          project_counter.items(), key=lambda x: x[1], reverse=True)[:10]
+      log_string = ""
+      for tid, count in top_matched_testcase:
+        log_string += "%s: %d, " % (tid, count)
```

Comment:
nit: please use f-strings. i'll fix this to get this deployed today.

Reply:
Thanks!